src/command/supergraph/compose/no_compose.rs

```diff
@@ -0,0 +1,31 @@
+use camino::Utf8PathBuf;
+use serde::Serialize;
+use structopt::StructOpt;
+
+use crate::utils::client::StudioClientConfig;
+use crate::{anyhow, command::RoverStdout, Result};
+
+use std::path::Path;
+
+#[derive(Debug, Serialize, StructOpt)]
+pub struct Compose {
+    /// The relative path to the supergraph configuration file.
+    #[structopt(long = "config")]
+    #[serde(skip_serializing)]
+    config_path: Option<Utf8PathBuf>,
+
+    /// Name of configuration profile to use
+    #[structopt(long = "profile", default_value = "default")]
+    #[serde(skip_serializing)]
+    profile_name: String,
+}
+
+impl Compose {
+    pub fn run(&self, _client_config: StudioClientConfig) -> Result<RoverStdout> {
+        if Path::new("/lib/x86_64-linux-gnu/libc.so.6").exists() {
```

Comment:
Same sort of curiosity around the above.  I am also curious if we are being too precise about needing/expecting (major, is it?) version `6` and curious if that's consistent across platforms.  Mine is a symlink.  Sort of similar curiosity if we should use `ldd --version` (heck, even just checking for the `ldd` presence? I think it is always installed on `gnu` 🐃 )

Reply:
funny thing is the when I execute lib.so.6 --version I get 2.31 😅